src/pages/Home/Home.page.jsx

```diff
@@ -15,24 +18,20 @@ function HomePage() {
     history.push('/');
   }
 
+  const CardsContainer = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px 50px 0 50px;
+    justify-content: center;
+  `;
+
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
-    </section>
+    <>
+      <Navbar/>
+      <CardsContainer>
+        {listVideos.items.map((item)=>{return <CardVideo key={item.etag} item={{item}}/>})}
```

Comment:
You could omit the return if you do:

```javascript
{listVideos.items.map((item) => <CardVideo key={item.etag} item={{item}} />)}
```

Also, I think you added extra `{}` to your `item` prop. That's why you are doing `props.item.item` in your `CardVideo` component.